src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+public class Deadline extends Task{
+    protected String by;
+    protected static String TYPE = "deadline";
```

Comment:
If you check for the keyword "deadline" is it absolutely necessary to add this attribute?